components/GlobalStyle/sub-components/Typography.jsx

```diff
@@ -90,9 +90,17 @@ ul, ol {
 const Typography = () => (
   <>
     <Style />
+    <link
+      crossOrigin="anonymous"
+      rel="preload"
+      href="https://fonts.googleapis.com/css?family=Nunito+Sans:400,600,700,400i,600i,700i&display=swap"
+      as="font"
```

Comment:
Divergence with the href

Reply:
The Font Attribute already references the CSS/@font-face type, which is used in this case for the google font.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#attr-type

![image](https://user-images.githubusercontent.com/37362664/220664893-d2f30a2f-8c1a-42bb-b280-e2cfb7d72b57.png)

Reply:
Very good!